internal/providers/terraform/azure/registry.go

```diff
@@ -102,6 +102,7 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 	GetAzureRMWindowsVirtualMachineScaleSetRegistryItem(),
 	getAzureRMVPNGatewayRegistryItem(),
 	getAzureRMVPNGatewayConnectionRegistryItem(),
+	getAzureSQLManagedInstanceRegistryItem(),
```

Comment:
this isn't well documented - but this `ResourceRegistry` slice is actually alphabetically ordered. Do you mind moving this function to the correct place.